storage-management/storage-management.md

```diff
@@ -0,0 +1,85 @@
+
+**Storage management in Microsoft 365 isn't just a routine task - it's a strategic necessity.** Your monthly expenses are directly linked to the efficiency of your storage management, particularly within SharePoint sites connected to your organization's purchased licenses.
+
+After you run out of storage for your SharePoint sites, depending on your plan, you can use the Office 365 Extra File Storage add-on to buy more storage in 1GB increments.
+
+However, **the best thing you can do to avoid additional costs is to clean up your storage proactively** and use tools that are available to manage and optimize the usage of your storage.
+
+* [Read this article provided by Microsoft to learn more about SharePoint storage limits.](https://learn.microsoft.com/en-us/office365/servicedescriptions/sharepoint-online-service-description/sharepoint-online-limits)
+
+* [To find more information and see if your plan is eligible to buy extra storage, explore the following Microsoft article.](https://learn.microsoft.com/en-us/microsoft-365/commerce/add-storage-space?view=o365-worldwide)
+
+## Get Started with Storage Management
+
+To start with your Storage Management, make sure it's configured based on your preferences within Syskit Point.
+
+That's where you can set up certain metrics that help you with the clean-up of your storage, such as how many versions of a file you want to keep or how old files should be before you delete them. 
+
+If you haven't done it already, [the first step is to configure your environment for Storage Management, take a look at this article.](../configuration/configure-point-for-storage-management.md)
+
+Next, for a step-by-step guide, [take a look at the Configure Storage Management within Syskit Point article.](../../configuration/configure-storage-management.md)
+
+Once you're done configuring storage management, it's time to configure your tenant storage limit policy.
+
+Syskit Point provides a predefined Tenant Storage Limit policy that is applied tenant-wide and cannot be deleted. However, it can also be customized to your preferences. This policy is used to detect and monitor your storage, but if you choose to, you can also enable task delegation.  
+
+For more details and instructions, [read through the Configure Tenant Storage Limit Policy article.](../governance-and-automation/automated-workflows/tenant-storage-admin.md)
+
+Once the policy has been edited based on your preferences, you can [take a look at the Tenant Storage Security & Compliance Check article](../governance-and-automation/security-compliance-checks/tenant-storage.md) to see all the details and information you can get from this policy being detected in your environment.
+
+Now that we've covered how to begin utilizing your storage management within Syskit Point let's look into the valuable information and actions that can be completed from the Storage Metrics reports.
+
+### An Overview of the Storage Metrics Reports
+
```

Comment:
ako gore u bulletima linkamo na odvojene članke i stavimo neki kratki opis, nisam siguran dal nam onda i ovdje treba dodatno još jednom opis reporta

Reply:
Ja bi report section ostavila odvojeno od ovih checks gore. U report clanku bas nemamo screenove i drukcije su organizirane informacije, tako da bi ostavila ovo kako je kao neki overview kako to izgleda. napravit cu sad PR sa promjenama pa vidi sta mislis, ako jos uvijek mislis da treba stavit samo link na reporte i maknut tu sekciju mogu to napravit. :)